src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/IActionInvokerFactory.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Abstractions;
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
```

Comment:
I didn't see any product code that needed to change as a result of this. Most things that reference this will already reference this namespace because the action selector is also here.